app/src/app/components/sidebar/ColorPicker.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import { palette, color10 } from "../../constants/colors";
+import { Button } from "@radix-ui/themes";
+import { styled } from "@stitches/react";
+import * as RadioGroup from "@radix-ui/react-radio-group";
+import { blackA } from "@radix-ui/colors";
+import { useMapStore } from "../../store/mapStore";
+
+export function ColorPicker() {
+  const [color, setColor] = useState(null);
+  const [open, setOpen] = useState(false);
+  const {
+    selectedZone,
+    setSelectedZone,
+    setZoneAssignments,
+    accumulatedGeoids,
+  } = useMapStore((state) => ({
+    selectedZone: state.selectedZone,
+    setSelectedZone: state.setSelectedZone,
+    setZoneAssignments: state.setZoneAssignments,
+    accumulatedGeoids: state.accumulatedGeoids,
+  }));
+  const colorArray = color10;
+  if (!colorArray) return null;
+  const handleRadioChange = (value) => {
+    console.log(
+      "setting accumulated geoids to old zone",
+      selectedZone,
+      "new zone is",
+      value
+    );
+    setZoneAssignments(selectedZone, accumulatedGeoids);
+    setSelectedZone(value);
```

Comment:
H: We're setting the selected zone to a hex color? Strongly thing we should set the zones to integers. The colors could get reshuffled and mess this up. Worst case if you're using integers the map will just look different. And the backend is expecting integer zones.

Reply:
Any ints will work. Can do 1-indexed or 0-indexed